spec/queries/follow_ups_query_spec.rb

```diff
@@ -129,6 +129,19 @@
                                   second_follow_up_date => 1})
         end
 
+        it "counts encounters created today when last is supplied and Groupdate time zone is not UTC" do
+          with_reporting_time_zone do
```

Comment:
Can you also freeze time here? Otherwise this will fail around day boundaries for certain time zones.